cmd/depsyncer/main.go

```diff
@@ -31,7 +31,7 @@ func main() {
 		flDebug   = flag.Bool("debug", false, "log debug messages")
 		flADebug  = flag.Bool("debug-assigner", false, "additional debug logging of the device assigner")
 		flStorage = flag.String("storage", "", "storage backend")
-		flDSN     = flag.String("storage-dsn", "", "storage data source name")
+		flDSN     = flag.String("storage-dsn", "file", "storage data source name")
```

Comment:
I think this meant to be on the `flStorage` flag?

Reply:
Oops